crates/ruff/src/rules/flake8_bandit/rules/shell_injection.rs

```diff
@@ -173,6 +173,16 @@ impl From<&Keyword> for Truthiness {
                     Truthiness::Truthy
                 }
             }
+            ExprKind::Constant {
+                value: Constant::Str(str),
+                ..
+            } => {
+                if str.is_empty() {
+                    Truthiness::Falsey
+                } else {
+                    Truthiness::Truthy
+                }
+            }
```

Comment:
I guess we should also include `JoinedStr` here, and maybe the `ITERABLE_INITIALIZERS` logic we see in `flake8-pytest`? Whatever's most comprehensive.

Reply:
Done.